bot/cogs/weather.py

```diff
@@ -0,0 +1,105 @@
+"""Commands for the weather module."""
+import discord
+from discord.ext import commands
+import aiohttp
+from bot.settings import WEATHER_TOKEN
+
+
+class Weather(commands.Cog, name='weather'):
```

Comment:
By our convention, the `name` should be capitalized, so you can remove the `name` kwarg entirely as your class name is already good.